android/keyple-plugin/android-nfc/src/main/kotlin/org/eclipse/keyple/plugin/android/nfc/AndroidNfcPluginImpl.kt

```diff
@@ -37,11 +40,18 @@ internal object AndroidNfcPluginImpl : AbstractPlugin(AndroidNfcPlugin.PLUGIN_NA
         Timber.d("InitNativeReader() add the unique instance of AndroidNfcReaderImpl")
         val readers = ConcurrentHashMap<String, Reader>()
         readers[AndroidNfcReader.READER_NAME] = if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N) {
-            AndroidNfcReaderPreNImpl
+            AndroidNfcReaderPreNImpl(activity)
```

Comment:
Can we now have multiple instances of NFC readers ?

Reply:
We can have multiple instances of NFC reader (using the same activity) but only the last to call 'enableReaderMode()' will be notified when a new tag is discovered

Reply:
What is the purpose of havinfg instance of the reader not being able to start a detection?